--- orig/lib/extoption.ml
+++ mod/lib/extoption.ml
@@ -8,15 +8,15 @@
 
 (** Is this option none? *)
 let is_none = function
-	  None -> true
+	None -> true
 	| _ -> false
 
 (** Is this option some? *)
 let is_some = function
-	  Some(x) -> true
+	Some(x) -> true
 	| _ -> false
 
 (** Get an option value. *)
-let get_option = function
-	  Some(x) -> x
+let get_option o = function
+	Some(x) -> x
 	| None -> raise Empty_option
