--- orig/net/socketping.py
+++ mod/net/socketping.py
@@ -25,7 +25,7 @@
 
     return s
 
-def waitForThem(sockets, timeout, h):
+def waitForThem(sockets, timeout):
     mysockets = list(sockets)
     found=[]
     finished = time.time() + timeout
@@ -39,7 +39,6 @@
         for s in r:
             try:
                 c=s.recv(1)
-                print "Found %s:%d" % h[s]
                 found.append(s)
             except socket.error, e:
                 pass
@@ -50,7 +49,6 @@
             if s in mysockets:
                 try:
                     if s.send("x") == 1:
-                        print "Found %s:%d" % h[s]
                         found.append(s)
                 except socket.error, e:
                     pass
@@ -66,15 +64,13 @@
     rv=0
     timeout=float(sys.argv[1])
     stuff=[x.split(':') for x in sys.argv[2:]]
-    h={}
     sockets=[]
     for a, b in stuff:
         s=getSocket(a, int(b))
-        h[s]=(a, int(b))
         sockets.append(s)
 
     # sockets=[getSocket(a, int(b)) for a, b in stuff]
-    found=waitForThem(sockets, timeout, h)
+    found=waitForThem(sockets, timeout)
 
     return rv
 
